backend/src/handlers/file_map_operations.py

```
@@ -6,14 +6,17 @@
 import logging
 import traceback
 from typing import Dict, Any, List, Optional
+import uuid
 from botocore.exceptions import ClientError
 from pydantic import ValidationError
 from datetime import datetime, timezone
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import

(F401)